email-mailjet/src/test/groovy/io/micronaut/email/mailjet/MailjetEmailSenderSpec.groovy

```diff
@@ -23,7 +23,7 @@ class MailjetEmailSenderSpec extends Specification {
                 env["MAILJET_API_SECRET"] &&
                 env["GMAIL_USERNAME"] &&
                 env["GMAIL_PASSWORD"] &&
-                (!CiUtils.runningOnCI() || (CiUtils.runningOnCI() && jvm.isJava11()))
+                (!CiUtils.runningOnCI() || (CiUtils.runningOnCI() && jvm.isJava17()))
```

Comment:
I think we did it not to avoid running this functional test for every java matrix entry. Running too many time the test was exhausting the free monthly emails offered by Mailjet.